images/jdk/image.yaml

```diff
@@ -14,11 +14,12 @@ versions:
   - apko:
       config: configs/latest-dev.apko.yaml
       extractTagsFrom:
-        package: openjdk-11
+        package: openjdk-17
         prefix: openjdk-
```

Comment:
like the other comment, this is going to overwrite latest above. should just remove `extractTagsFrom`. This was already broken before your changes it seems

Reply:
updated